packages/client/src/intros/shared/Intro.tsx

```diff
@@ -28,22 +29,21 @@ export default ({
   usableForBullets = [],
   usableForText,
 }: Props) => {
+  const { t } = useTranslation();
   const influence =
     dependantOnSituation && dependantOnQuestions
-      ? "Uw situatie en uw antwoorden bepalen of u een omgevingsvergunning nodig hebt."
+      ? t("introPage.common.first situation")
       : dependantOnQuestions
-      ? "Uw antwoorden bepalen of u een omgevingsvergunning nodig hebt."
+      ? t("introPage.common.second situation")
```

Comment:
Can you change the key to `introPage.common.situation dependent on questions only`?